src/print.rs

```diff
@@ -104,50 +105,71 @@ pub fn explain(args: ArgMatches) {
             let value = module.get_segments().join("");
             ModuleInfo {
                 value: ansi_term::ANSIStrings(&module.ansi_strings()).to_string(),
-                value_len: value.chars().count() + count_wide_chars(&value),
+                value_len: better_width(value.as_str()),
                 desc: module.get_description().to_owned(),
             }
         })
         .collect::<Vec<ModuleInfo>>();
 
-    let mut max_ansi_module_width = 0;
-    let mut max_module_width = 0;
-
-    for info in &modules {
-        max_ansi_module_width = std::cmp::max(
-            max_ansi_module_width,
-            info.value.chars().count() + count_wide_chars(&info.value),
-        );
-        max_module_width = std::cmp::max(max_module_width, info.value_len);
-    }
+    let max_module_width = modules.iter().map(|i| i.value_len).max().unwrap_or(0);
 
     let desc_width = term_size::dimensions()
         .map(|(w, _)| w)
-        .map(|width| width - std::cmp::min(width, max_ansi_module_width));
+        // In addition to the module width itself there are also 6 padding characters in each line.
+        // This needs to be updated here and later in the function on line formatting changes.
```

Comment:
Sorry didn't realise this earlier but if the 6 is used here are later on should we use a constant for this so they don't end up getting out of sync?

Reply:
I've replaced it with a constant.